cli/users.go

```diff
@@ -106,6 +106,8 @@ func (hbc *HoneybadgerClient) DeleteUser(userID int, teamID int) error {
 
 // GetUserFromTeams - Get User information from Teams
 func (hbc *HoneybadgerClient) GetUserFromTeams(userEmail string) (userTeams []HoneybadgerUser, err error) {
```

Comment:
I don't get your point of view.

The map is used to only iterate if the user is not in member team. As a consequence, I need to check if the user is inside invitations array.

Reply:
Yes, but the `userEmail` is passed as parameter in the function, so imagine the following case:
* User `marcel.puchol@sequra.es` belongs to two teams (team `A` and team `B`) and has been invited in the team `C`
* In the first iteration, you will store `insertedUser["marcel.puchol@sequra.es"] = true`
* Then, once you will check the conditional `insertedUser["marcel.puchol@sequra.es"]`, it will be true, so I guess that you should check if the user is present in a concrete team. Being this the case and only being possible to be present in a team with a concrete email (it means, in the team `A`, the email "marcel.puchol@sequra.es" should only be present once, you can improve the loop by the previous proposal

Reply:
Yes, you're right in what you're saying. However, I would like to use the benefits of a `map` instead of `continue`. 

Also, I'll add these corner cases into unit test 😄